app/ui-message/client/message.html

```diff
@@ -51,9 +51,9 @@
 							<i class="icon-mail" title='{{_ "Message_sent_by_email"}}' aria-label='{{_ "Message_sent_by_email"}}'></i>
 						</span>
 					{{/if}}
-					{{#if edited}}
+					{{#if msg.editedAt}}
 						<span class="edited" title='{{_ "edited"}} {{_ "at"}} {{editTime}} {{_ "by"}} {{editedBy}}'>
-							<i class="icon-pencil{{#if $neq editedBy msg.u.username}} error-color{{/if}}" aria-label="{{_ "Edited"}}"></i>
+							<i class="icon-pencil{{#if $neq msg.editedBy.username msg.u.username}} error-color{{/if}}" aria-label="{{_ "Edited"}}"></i>
```

Comment:
Again, try to fix the existing helpers instead of getting the data directly from the `msg` object